### DISCUSSION TITLE
Entity status translation for custom sentences

### DISCUSSION THREAD

**Discussion by llluis** [OHF-Voice/intents#1869] [_General]
Hello! 

While looking for missing pt-br intents translation (and finding out there are already PRs for that), I stumbled on @tetele's post https://github.com/home-assistant/intents/discussions/1768 indicating "state [is translated](https://github.com/home-assistant/core/blob/fb615817b4ccb3a0dcfeaf0384a1ff29633170cd/homeassistant/components/conversation/default_agent.py#L335) before being used in the intent response."

However, that doesn't seem to apply for custom intents.
I was testing ```HassGetState``` (as per [Custom Sentences](https://www.home-assistant.io/voice_control/custom_sentences/#customizing-responses)) and the response ```{{ state.state_with_unit }}``` is returned in English (on or off, in my test, if asking for a light).

Is this the expected behaviour?

**Comment by llluis** [comment]
Did another test with the ```intent_script``` and the result is the same: entity status is in English.

**Comment by tetele** [comment]
That might have to do with a lack of translations for the specific state that you're getting, in the specific language you're using. In the HA interface, is the same state translated?
